archetypes/Exchange/TokenHoldingsTable/index.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useMemo } from 'react';
+import { HiddenExpand } from '@components/General';
+import { Logo, tokenSymbolToLogoTicker } from '@components/General';
+import styled from 'styled-components';
+import { SideEnum } from '@libs/constants';
+import useEscrowHoldings from '@libs/hooks/useEscrowHoldings';
+import { PoolInfo } from '@context/PoolContext/poolDispatch';
+
+interface FeeNoteProps {
+    pool: PoolInfo['poolInstance'];
+    side: SideEnum;
+    userBalances: PoolInfo['userBalances'];
+}
+
+const TokenHoldingsTable: React.FC<FeeNoteProps> = ({ pool, side, userBalances }) => {
+    const escrowRows = useEscrowHoldings();
+
+    const filteredEscrow = useMemo(() => escrowRows.filter((v) => v.poolAddress === pool.address), [pool]);
+
+    const isLong = side === SideEnum.long;
+    const claimableType = isLong ? 'claimableLongTokens' : 'claimableShortTokens';
+    const oppositeClaimableType = isLong ? 'claimableShortTokens' : 'claimableLongTokens';
+    const escrowBalance = filteredEscrow?.[0]?.[claimableType]?.balance?.toNumber();
+    const oppositeEscrowBalance = filteredEscrow?.[0]?.[oppositeClaimableType]?.balance?.toNumber();
+
+    const [tokenBalance, oppositeTokenBalance] = useMemo(
+        () =>
+            isLong
+                ? [userBalances.longToken, userBalances.shortToken]
+                : [userBalances.shortToken, userBalances.longToken],
+        [isLong, userBalances.longToken, userBalances.shortToken],
+    );
+
+    const [currentToken, oppositeCurrentToken] = useMemo(
+        () => (isLong ? [pool.longToken, pool.shortToken] : [pool.shortToken, pool.longToken]),
+        [isLong, pool.longToken, pool.shortToken],
+    );
+
+    return (
+        <StyledHiddenExpand defaultHeight={0} open>
+            <Table>
```

Comment:
There's a `TWTable` in components/General

Reply:
I used it initially but found myself having to override a lot of things and using !important too much

Not sure it was the right approach but it can be taken into consideration when we move TWTable to styled components